### PR TITLE
chore: remove 1.4.43 azure cni binaries

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by jaer-tsun — APPROVED** [review]
lgtm

**Review by cameronmeissner — APPROVED** [review]

**Review by rbtr — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -48,7 +48,6 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/cni-dropgz:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.0.4",
         "v0.0.4.1",
         "v0.0.9"
```
Comment (jaer-tsun) [inline vhdbuilder/packer/components.json:52]: looks like we can replace v0.0.9 -> v0.0.13

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -347,8 +345,7 @@
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
         "1.5.5",
```
Comment (jaer-tsun) [inline vhdbuilder/packer/components.json:347]: we can replace v1.5.5 -> v1.5.11

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (rbtr) [reply vhdbuilder/packer/components.json:52]: not .13, should be .11

Comment (anujmaheshwari1) [reply vhdbuilder/packer/components.json]: done

File: vhdbuilder/packer/components.json (hunk 2 above)
Comment (anujmaheshwari1) [reply vhdbuilder/packer/components.json]: done

**Review by jaer-tsun — APPROVED** [review]
lgtm

**Pull request merged**